Dockerfile

```diff
@@ -29,12 +29,9 @@ ENV HADOOP_VERSION 3.2.1
 ENV PATH /opt/hadoop-${HADOOP_VERSION}/bin:/miniconda/envs/sqlflow-dev/bin:/miniconda/bin:/usr/local/go/bin:/go/bin:$PATH
 ENV IPYTHON_STARTUP /root/.ipython/profile_default/startup/
 
-# Main steps to Build
-COPY . ${GOPATH}/src/sqlflow.org/sqlflow
```

Comment:
`COPY` the whole directory will cause the caching fails because the directory always changes in a Pull Request. Copy only the build script so that all environment builds are cached.